### PR TITLE
Update ec

### PR DESCRIPTION
**Pull request by james9909**
Currently updating only names, since some people have yet to send me their photos. Will update later when they do.

### CODE REVIEW

File: _data/ec.yml [hunk 1]
```diff
@@ -2,52 +2,46 @@
...
 lab-ops:
+    name: Arpita Abrol
+    photo: /img/ec/arpita.jpg
+ops:
     name: Dana Yang
     photo: /img/ec/dana.jpg
 finance:
-    name: Sally Ko
-    photo: /img/ec/sally.jpg
+    name: Winne Lin
```
Comment (justinkim) [inline _data/ec.yml]: Winnie

Comment (james9909) [inline _data/ec.yml]: It's Winne

File: _data/ec.yml [hunk 2]
```diff
@@ -2,52 +2,46 @@
...
-    photo: /img/ec/leith.jpg
+    name: Piotr Milewski
 design:
-    name: Shantanu Jha
-    photo: /img/ec/shantanu.jpg
+    name: Vacant
 machinist:
     name: Anya Keller
     photo: /img/ec/anya.jpg
-animation:
```
Comment (justinkim) [inline _data/ec.yml]: Has the position been removed? If not, please ask animation who their director is.

File: _data/ec.yml (hunk 1 above)
Comment (WilsonBerkow) [inline _data/ec.yml]: No it's spelled with no "i". Catches a lot of people off guard.

Comment (justinkim) [inline _data/ec.yml]: my b

File: _data/ec.yml (hunk 2 above)
Comment (justinkim) [inline _data/ec.yml]: :clap:

**Comment by justinkim** [comment]
lgtm

**Pull request merged**